### PR TITLE
Translation of 2020 accessibility chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

### CODE REVIEW

File: src/content/ja/2020/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+  description="バーチャートを見ると、デスクトップサイトの16.0％、モバイルサイトの19.7％が拡大縮小を無効にしており、デスクトップサイトの21.1％、モバイルサイトの27.1％がズームの上限を1に設定しておりデスクトップサイトの24.4％、モバイルサイトの30.7％がこれらのいずれかを設定していることがわかります。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSjkB_XAYiwkYrMuoXp44mdqMGJVDUkTr_48ELozY72Mdv3OlxeWV9ysbDY9bs6hA7LnJTrHar9aZlM/pubchart?oid=1053957382&format=interactive",
+  sheets_gid="1095274901",
+  sql_file="viewport_zoom_scale.sql"
+  )
+}}
+
+デスクトップページの29.34%とモバイルページの30.66%が、`maximum-scale`を1未満の値に設定するか、`user-scalable`を`0`または`none`に設定することで、スケーリングを無効にしようとしていることがわかった。いくつかのOSでは、HTMLで設定されているズームや拡大縮小の無効化に対応していないものがあります。これに対応しているシステムでは、一部のシステムではページが事実上使えなくなることがあります。ブラウザのズームを無効にするのを避ける理由については、Adrian Roselliの記事[Don't Disable Zoom](https://adrianroselli.com/2015/10/dont-disable-zoom.html) を参照してください。
+
+### Language identification
```
Comment (bazzadp) [inline src/content/ja/2020/accessibility.md:88]: Missed a translation?

The irony of missing *this* translation - out of all of them 🤣

File: src/content/ja/2020/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+
+フォームは、アクセシビリティの観点から、最も重要なものの一つです。フォーム入力を正しく送信することは、ユーザーがウェブサイトやアプリケーションの主要な操作を実行できることを意味します。例えば、登録フローにアクセスできない場合、障害のあるユーザーはサイトに全くアクセスできないかもしれません。
+
+デジタル・アクセシビリティは市民権であり、すべての人が情報にアクセスし、ウェブ上で同じ機能を実行する平等な権利を持っていることを忘れてはなりません。障害のあるユーザーが、特に政府サービスやその他の重要な活動のためにフォームを提出するような作業のために、ウェブの主要なタスクの実行や情報へのアクセスを妨げられた場合、民間部門と公共部門の両方で差別の対象となることは明らかです。
+
+### フォームの検証
+
+どのようなフォームのエラー処理であっても、支援技術に伝えることは非常に重要です。これを処理するためには、バリデーションの実装に応じてさまざまなテクニックがあります。Web AIMの[Usable and Accessible Form Validation and Error Recovery](https://webaim.org/techniques/formvalidation/) の記事は、様々なアクセシブルなフォームバリデーション戦略について学ぶための素晴らしいリソースです。
+
+フォーム要素が必要とされる場合、これも支援技術に伝える必要があります。ネイティブHTMLフォーム要素には[`required`属性](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required) を使用し、カスタマイズされた要素には [`aria-required`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute) 属性が必要になるかもしれません。フォーム送信に問題がある場合は、その旨を支援技術に伝える必要があります。
```
Comment (bazzadp) [inline src/content/ja/2020/accessibility.md:517]: These are available in Japanese:

```suggestion
フォーム要素が必要とされる場合、これも支援技術に伝える必要があります。ネイティブHTMLフォーム要素には[`required`属性](https://developer.mozilla.org/ja/docs/Web/HTML/Attributes/required) を使用し、カスタマイズされた要素には [`aria-required`](https://developer.mozilla.org/ja/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute) 属性が必要になるかもしれません。フォーム送信に問題がある場合は、その旨を支援技術に伝える必要があります。
```

File: src/content/ja/2020/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+  - 障害者実務者を含むアクセシビリティスキルを持った人材の採用・採用。
+  - 障がいのある人にも対応した包括的な職場環境づくり
+
+**UX/プロダクトデザイナー**
+  - 調査・構想の段階で様々な障害を持つ人たちのことを考え、話をすること。
+  - 見出しの階層構造、スキップリンク、代替テキストの提案（コピーライターやコンテンツ担当者によるものもあります）、スクリーンリーダー専用テキストなどのアクセシビリティ情報をワイヤーフレームに注釈をつけます。
+
+**UIデザイナー**
+  - 色のコントラストの選択、フォントの選択、間隔、行の高さの考慮事項。
+  - アニメーションの考察（必要かどうかの判断、[`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)シナリオへの静的アセットの供給、ポーズ/ストップ機構の設計）。
```
Comment (bazzadp) [inline src/content/ja/2020/accessibility.md:561]:
```suggestion
  - アニメーションの考察（必要かどうかの判断、[`prefers-reduced-motion`](https://developer.mozilla.org/ja/docs/Web/CSS/@media/prefers-reduced-motion)シナリオへの静的アセットの供給、ポーズ/ストップ機構の設計）。
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks good technically. A few small things I noticed.

File: src/content/ja/2020/accessibility.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2020/accessibility.md]: Misstake 😅

File: src/content/ja/2020/accessibility.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2020/accessibility.md]: Thanks, I forget check url.

File: src/content/ja/2020/accessibility.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2020/accessibility.md]: Thanks, I forget check url.

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Not sure if @MSakamaki or @hi-toshi are going to get a chance to review this so going to go ahead and merge this as going a release. Would love to hear you feedback if you do get a chance to review and can fix in a future PR.

Thanks for working on this @ksakae1216 !

**Pull request merged**